requests_toolbelt/utils/deprecated.py

```diff
@@ -32,7 +32,8 @@ def get_encodings_from_content(content):
     :type content: bytes
```

Comment:
I have fixed this problem.
If we do not change it, get_unicode_from_response will be break in Python3. Because, bytes.decode(encoding) method or str(bytes, encoding) can not accept a bytes encoding string, it must be a str.